acceptance/setup/aio/010_Install.rb

```diff
@@ -11,7 +11,9 @@
   repo_configs_dir = 'repo_configs'
   logger.debug('about to install repo for puppet-agent from ' + sha.to_s + ' ' + repo_configs_dir.to_s)
   hosts.each do |host|
-    install_repos_on(host, 'puppet-agent', sha, repo_configs_dir)
+    if !host.platform.start_with?('windows')
+      install_puppetlabs_dev_repo(host, 'puppet-agent', sha, repo_configs_dir, { :dev_builds_repos => 'PC1'})
+    end
```

Comment:
We should be able to use `install_puppet_agent_dev_repo` on all hosts (in beaker 2.23.0 and up), and it will do the right thing for all agent platforms. There's some additional complexity if you are installing a master though: https://github.com/puppetlabs/puppet/blob/master/acceptance/setup/aio/pre-suite/010_Install.rb#L9-L18